### PR TITLE
y-aoyama.jp

### PR DESCRIPTION
**Pull request by gork7777**
### To avoid invalid pull requests, please check and confirm following terms

- [x] This is not an ad/bug report;
- [x] My code follows the [guidelines](https://github.com/AdguardTeam/AdguardFilters/blob/master/CONTRIBUTING.md) and [syntax](https://kb.adguard.com/general/how-to-create-your-own-ad-filters) of this project;
- [x] I have performed a self-review of my own changes;
- [x] My changes do not break web sites, apps and files structure.

### If the problem does not fall under any category that is listed here, please write a comment below in corresponding section

- [ ] Missed ads or ad leftovers;
- [ ] Website or app doesn't work properly;
- [ ] AdGuard gets detected on a website;
- [ ] Missed analytics or tracker;
- [x] Social media buttons — share, like, tweet, etc;
- [x] Annoyances — pop-ups, cookie warnings, etc;
- [ ] Filters maintenance.

### Enter the issue address
https://github.com/AdguardTeam/AdguardFilters/issues/124006

### Add your comment and screenshots

1. Your comment

Also blocked support chat.

### Terms

- [x] By submitting this issue, I agree that pull request does not contain private info and all conditions are met

### CODE REVIEW

File: AnnoyancesFilter/sections/widgets.txt [hunk 1]
```diff
@@ -301,6 +301,7 @@ full-count.jp#?#section > div.c-heading:has(> p.c-heading__en:contains(RECOMMEND
 .ru/proxy_*.php?p___=*&part_url=*.php?callback=*&token=*&iso=
 .net/proxy_*.php?p___=*&part_url=*.php?callback=*&token=*&iso=
 !
+||whatya.solairo-api.com/aoscs/chat.js
```
Comment (Yuki2718) [inline AnnoyancesFilter/sections/widgets.txt:304]: Apparently the whole domain is for chat so `||whatya.solairo-api.com^$third-party` will be okay.

**Review by Yuki2718 — CHANGES_REQUESTED** [review]
Add `||tag.voice.zetacx.net^` to tracking server.

**Review by Yuki2718 — APPROVED** [review]

File: AnnoyancesFilter/sections/widgets.txt (hunk 1 above)
Comment (Alex-302) [reply AnnoyancesFilter/sections/widgets.txt]: Not sure about that. As I see, that bot provides useful info.

**Review by zloyden — APPROVED** [review]

File: AnnoyancesFilter/sections/widgets.txt (hunk 1 above)
Comment (Yuki2718) [reply AnnoyancesFilter/sections/widgets.txt]: Well, most of online consultant widgets have some utility. What is the criteria to block or not to block?

Comment (Alex-302) [reply AnnoyancesFilter/sections/widgets.txt]: If they really annoying. We do not block literally all widgets with this purpose.
We will change the policy for annoyances a bit later.

Comment (Alex-302) [reply AnnoyancesFilter/sections/widgets.txt]: After a couple of days, we'll split the Annoyances filter apart and the problem will be less serious.

**Pull request merged**